### ISSUE TITLE
ExceptionInInitializerError when Running emoji-java using the code

### ISSUE THREAD

**Issue by sanjrockz** [vdurmont/emoji-java#10]
I was trying to run your application using the source code and it keeps throwing me ExceptionInInitializerError. Here's the exception I see.
Exception in thread "main" java.lang.ExceptionInInitializerError
	at com.vdurmont.emoji.EmojiParser.parseToAliases(EmojiParser.java:53)
	at com.vdurmont.emoji.EmojiParser.parseToAliases(EmojiParser.java:28)
	at com.vdurmont.emoji.MainClass.main(MainClass.java:8)
Caused by: java.lang.RuntimeException: java.lang.NullPointerException: source
	at com.vdurmont.emoji.EmojiManager.<clinit>(EmojiManager.java:39)
	... 3 more
Caused by: java.lang.NullPointerException: source
	at java.util.Scanner.makeReadable(Scanner.java:629)
	at java.util.Scanner.<init>(Scanner.java:622)
	at com.vdurmont.emoji.EmojiLoader.inputStreamToString(EmojiLoader.java:45)
	at com.vdurmont.emoji.EmojiLoader.loadEmojis(EmojiLoader.java:33)
	at com.vdurmont.emoji.EmojiManager.<clinit>(EmojiManager.java:25)
	... 3 more

I'm passing the correct file path to emojis.json file, but still it fails to read from the stream. I could not find much info by debuging the code, so though of posting here to get your view on why this happens.
Thank you in advance!

**Comment by vdurmont** [comment]
When you say "run using the source code", what do you mean exactly?

It seems that the application cannot find the file at `/emojis.json`. Do you have the file on your classpath?

**Comment by sanjrockz** [comment]
I was running EmojiParser using the source code you provided. Well, first I didn't add the /emojis.json to the classpath. Instead I changed the PATH variable in EmojiManager, so that PATH points to the correct location of emojis.json file. That gave me the error. But when you asked whether I added emojis.json to classpath, I added it and tried. Then it worked. Thank you for the quick response. Closing this thread :)
